bliss/catalog.py

```diff
@@ -9,7 +9,7 @@
 from torch.nn import functional as F
 
 
-class TileCatalog(UserDict):
+class AbstractTileCatalog(UserDict):
```

Comment:
if you would like, you could make this an abstract class explicitly by inheriting from `abc.ABC` but I don't think it's necessary

Reply:
Yeah I worried that would make things too complicated, especially since I want to implement some shared functionality